pytensor/link/pytorch/dispatch/elemwise.py

```diff
@@ -11,9 +14,38 @@ def pytorch_funcify_Elemwise(op, node, **kwargs):
     scalar_op = op.scalar_op
     base_fn = pytorch_funcify(scalar_op, node=node, **kwargs)
 
-    def elemwise_fn(*inputs):
-        Elemwise._check_runtime_broadcast(node, inputs)
-        return base_fn(*inputs)
+    if isinstance(scalar_op, ScalarLoop):
+        # note: scalarloop + elemwise is too common
+        # to not work, but @1031, vmap won't allow it.
+        # Instead, we will just successively unbind
+        def elemwise_fn(*inputs):
```

Comment:
Can you add some comments explaining the logic? It's not immediately obvious to me how it is working

Reply:
ha, yeah. It is strange. Do we have a standard kind comment dump for this kind of thing?

Reply:
No, anything will already help. There's no problem with being overly verbose / ELI5

Reply:
Cool, I'll give it a go.